docs/declarative-customization/list-form-conditional-show-hide.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Show or hide columns in a list form
+description: Customize which columns to show or hide using a conditional formula in the list form by constructing a simple formula that are equations that perform conditional checks on values in a SharePoint list or library. 
+ms.date: 03/16/2020
+localization_priority: Priority
+---
+
+# Show or hide columns in a list or library form
+You can show or hide columns in a list or library form as an alternative to deleting. When you hide a column, it doesn't affect the column or the data in the column, as it would if you deleted it. To re-use the column, you can just show it again in the form.
+
+To show or hide a column in a list or library form:
+
+1. Go to the list or library for which you want to show or hide columns in the form
+
+2. If you are in a list:
+
+- Open an item to view the item details in the display form
+
+3. If you are in a document library:
+
+- Select a file
+- In the **Information Pane**
+    - Locate the **Properties** section
+    - Click **Edit all**
+
+4. At the top of the form, select **Edit Form > Edit columns**
+
+5. In the **Edit columns** pane, check (to show) or uncheck (to hide) the box for the column or columns as needed.
+
+> Note: If you want to re-arrange the order of the columns, either drag-and-drop the column name, or select the up or down arrow next to the column name as preferred.
+
+6. When you're finished, select **Save**.
+
+## Specify conditional formula to show or hide columns
+You can show or hide columns in a list form based on another column's value by specifying a formula that are equations that perform conditional checks on values in a SharePoint list or library. 
```

Comment:
You can show or hide columns in a list form based on another column's value by specifying a formula that are equations performing conditional checks on values in a SharePoint list or library.